nvm.sh

```diff
@@ -178,29 +178,29 @@ nvm_install_latest_npm() {
     NVM_IS_0_9=1
   fi
 
-  if [ $NVM_IS_0_6 -eq 1 ]; then
+  if [ ${NVM_IS_0_6} -eq 1 ]; then
```

Comment:
why is this a better style?

Reply:
I'm not going to match some arbitrary styleguide, especially google's (whose style i rarely agree with), especially one that offers no justification for why to do this.

Reply:
Will I need to remove this one though?

Reply:
Yes - I'm suggesting that curly-bracing non-quoted variables doesn't seem to have any justification I'm aware of, so I don't want any non-quoted variables curly braced until there's a justification for it beyond "google says".